### ISSUE TITLE
Expose overflowNeeded prop in Toolbar

### ISSUE THREAD

**Issue by Neeeko** [SAP/ui5-webcomponents-react#2291]
**Is your feature request related to a problem? Please describe.**
As part of a SAP SuccessFactors UXR project, we're using the `Toolbar` component to display a `Title` and a few action buttons.
- In desktop mode, we need to display two action buttons: an Edit button and a custom overflow button to display secondary actions in an `ActionSheet`.
<img width="1787" alt="Screenshot 2021-11-10 at 12 01 06" src="https://user-images.githubusercontent.com/7316793/141101822-306563a5-ac7f-4e22-9577-d5e0e478fe16.png">

- In mobile mode, we _always_ want to display the overflow button to display all the action buttons.
<img width="495" alt="Screenshot 2021-11-10 at 12 02 07" src="https://user-images.githubusercontent.com/7316793/141101888-42733bbe-3079-4632-a094-b57f295babc2.png">

**Describe the solution you'd like**
Expose the [`overflowNeeded` variable](https://github.com/SAP/ui5-webcomponents-react/blob/main/packages/main/src/components/Toolbar/index.tsx#L136) as a prop so that optionally, developers can implement their own logic to toggle the overflow button.

For example:
```jsx
const overflowNeeded = window.innerWidth < 768;

<Toolbar overflowNeeded={overflowNeeded}>
  <Title>Test</Title>
  <ToolbarSpacer />
  <Button>Action 1</Button>
  <Button>Action 2</Button>
</Toolbar>

```

**Additional context**
In the current code, here's where the overflowNeeded variable is defined: https://github.com/SAP/ui5-webcomponents-react/blob/main/packages/main/src/components/Toolbar/index.tsx#L136
This is a good default behavior but it would be nice to be able to toggle the overflow button as needed.

**Comment by Lukas742** [comment]
Hi @Neeeko 

sorry for the late reply, it has been a busy month.
Offering an `overflowNeeded` prop doesn't seem viable for us as it would mean to also disable the overflow logic and therefore the Toolbar wouldn't really be much more than a `FlexBox` with styling. 
We could instead try to make it possible that elements will not move to the overflow popover after a user defined index. 
I guess this approach would be similar (or maybe even the same - I'm not quite sure as the issue is a bit confusing) to this [issue](https://github.com/SAP/ui5-webcomponents-react/issues/2060).

How do you feel about this approach?

**Comment by Lukas742** [comment]
This feature is also included in [v0.131.28](https://github.com/SAP/ui5-webcomponents-react/blob/rel/0.131/CHANGELOG.md#013128-2021-12-21).

**Comment by Neeeko** [comment]
Thanks @Lukas742, I will update the team!
